internal/fluentbit/config/builder/config_builder_test.go

```diff
@@ -180,7 +180,7 @@ func TestMergeSectionsConfigCustomOutput(t *testing.T) {
     name             tail
     alias            foo
     db               /data/flb_foo.db
-    exclude_path     /var/log/containers/telemetry-fluent-bit-*_kyma-system_fluent-bit-*.log
+    exclude_path     
```

Comment:
Let's skip this if empty. Should be easily solvable by using `SectionBuilder.AddIfNotEmpty` in `createInputSection`.